spec/namespaces/snapshot.yaml

```diff
@@ -294,7 +294,7 @@ components:
             type: object
             properties:
               ignore_unavailable:
-                description: If `true`, the request ignores data streams and indexes in `indices` that are missing or closed. If `false`, the request returns an error for any data stream or index that is missing or closed.
+                description: If `true`, the request ignores data streams and indexes in `indexes` that are missing or closed. If `false`, the request returns an error for any data stream or index that is missing or closed.
```

Comment:
If this refers to a variable (indices) then quoting it will work, so I think this means `indices`.